### PR TITLE
chore (ai): increase id generator default size from 7 to 16.

### PR DESCRIPTION
**Pull request by shaper**

### CODE REVIEW

File: content/docs/08-troubleshooting/01-migration-guide/04-migration-guide-4-0.mdx [hunk 1]
```diff
@@ -62,3 +62,9 @@ Please use the `headers` and `body` parameters instead.
 ### Removed `name` from UI message
 
 The `name` property from the `Message` type has been removed.
+
+### Increased default size of generated IDs
+
+The [`generateId`](/docs/reference/stream-helpers/generate-id) function, as well
+as generators created with `createIdGenerator`, now generate 16-character IDs.
```
Comment (lgrammel) [inline content/docs/08-troubleshooting/01-migration-guide/04-migration-guide-4-0.mdx:69]: We have not exposed `createIdGenerator` yet in `ai`. Let's only mention `generateId` here. The `generateId` reference docs might also need to be updated.

**Comment by lgrammel** [comment]
The PR needs a changeset (major change).

File: content/docs/08-troubleshooting/01-migration-guide/04-migration-guide-4-0.mdx (hunk 1 above)
Comment (shaper) [reply content/docs/08-troubleshooting/01-migration-guide/04-migration-guide-4-0.mdx:69]: `createIdGenerator` looks like it is (and has been) exported for a bit at least, but it isn't mentioned in any docs. Updated docs here to remove mention.

Updated `50-generate-id.mdx` to note the new default as well.

File: .changeset/swift-kids-jam.md
Comment (shaper) [inline .changeset/swift-kids-jam.md:1]: Please double check this as it's my first time generating a changeset. As it was major I was informed I should provide more detail. However, the PR is small, so I just stuck with the PR title effectively.

**Review by lgrammel — APPROVED** [review]

**Comment by lgrammel** [comment]
Looks good. I'll make a few minor tweaks & land.

**Pull request merged**